client/src/business-logic/get-answer.js

```diff
@@ -4,8 +4,6 @@ export const accessAnswers = (i = 0) => {
   const data = findAll();
   const questionsObject = data[0];
   const questionsArray = Object.values(questionsObject);
-  console.log(questionsArray[1][0].answers[1].text);
```

Comment:
I think in the future we should use `find.js` function to choose the right id (just thoughts)

Reply:
yeah, I was trying to use find(), instead of findAll(), but could not figure out how to make it work